### PR TITLE
Enhance Test Coverage for @yamada-ui/layouts

### LINKED ISSUE #786: Enhance Test Coverage for `@yamada-ui/layouts`
related: #775

### Description
Enhance test coverage for components with less than 80% coverage to meet or exceed 80%.

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### PR DESCRIPTION
**Pull request by txxxxc**
Closes #786 

## Description

Enhance Test Coverage for @yamada-ui/layouts

## Is this a breaking change (Yes/No):

No

## Additional Information

To keep the cost per review small, we have not written tests for the [ZStack](https://github.com/yamada-ui/yamada-ui/blob/e620bd0fa5c7c39501c50c020fcd6fb0cdd5b447/packages/components/layouts/src/stack.tsx#L211) component.

### CODE REVIEW

File: packages/components/layouts/tests/layouts.test.tsx [hunk 1]
```diff
@@ -2,16 +2,37 @@ import { render, a11y, screen, fireEvent, waitFor } from "@yamada-ui/test"
...
+  test("passes a11y test", async () => {
+    await a11y(
+      <AspectRatio>
+        <img src="https://image.xyz/source" alt="placeholder" />
+      </AspectRatio>,
+    )
+  })
+
+  test("renders aspect ratio correctly", () => {
+    render(
```
Comment (koralle) [inline packages/components/layouts/tests/layouts.test.tsx:28]: There's no assertion, but what is this code testing?

File: packages/components/layouts/tests/layouts.test.tsx [hunk 2]
```diff
@@ -97,6 +118,39 @@ describe("<Divider />", () => {
   })
 })
 
+describe("<SimpleGrid />", () => {
+  test("renders SimpleGrid correctly", () => {
+    render(<SimpleGrid>GridSimple</SimpleGrid>)
```
Comment (koralle) [inline packages/components/layouts/tests/layouts.test.tsx:123]: There's no assertion, but what is this code testing?

File: packages/components/layouts/tests/layouts.test.tsx [hunk 3]
```diff
@@ -243,3 +297,22 @@ describe("<Grid />", () => {
     })
   })
 })
+
+describe("<GridItem />", () => {
+  test("renders grid correctly", () => {
+    render(<GridItem>GridItem</GridItem>)
```
Comment (koralle) [inline packages/components/layouts/tests/layouts.test.tsx:303]: There's no assertion, but what is this code testing?

File: packages/components/layouts/tests/layouts.test.tsx (hunk 1 above)
Comment (txxxxc) [reply packages/components/layouts/tests/layouts.test.tsx]: My apologies. There is such a test in this file. I thought such a test would be valuable.

File: packages/components/layouts/tests/layouts.test.tsx [hunk 4]
```diff
@@ -97,6 +110,39 @@ describe("<Divider />", () => {
...
+    )
+    expect(getByTestId("simpleGrid")).toHaveStyle({
+      gridTemplateColumns: "repeat(3, minmax(0, 1fr))",
+    })
+  })
+})
+
 describe("<Stack />", () => {
   test("renders stack correctly", () => {
     render(<Stack>Stack</Stack>)
```
Comment (koralle) [inline packages/components/layouts/tests/layouts.test.tsx:148]: There are no assertions here; what is being tested?

**Comment by koralle** [comment]
Could you switch from `getBy*` to `findBy*`? 
We're planning to establish Lint rules for test code using [eslint-plugin-testing-library](https://github.com/testing-library/eslint-plugin-testing-library) soon, and we intend to enforce the use of `findBy*`.

File: packages/components/layouts/tests/layouts.test.tsx (hunk 1 above)
Comment (koralle) [reply packages/components/layouts/tests/layouts.test.tsx]: I should be the one apologizing for the oversight. Sorry about that. Your way of thinking is not wrong.

We'll start gradually fixing similar existing test code over here as well.

Thank you!

File: .changeset/tender-bears-confess.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (koralle) [inline .changeset/tender-bears-confess.md:1]: It is not necessary to run `pnpm changeset` for a PR that only adds or updates tests.  
Could you remove this file, please?

See [wiki](https://github.com/yamada-ui/yamada-ui/wiki/Tips-for-Test#frequently-asked-questions)

**Comment by txxxxc** [comment]
@koralle 
Thanks for the review, I just replaced the `getBy*` method with a `findBy*` method. For now, I just modified the code I added in the previous commit, and I would like to replace the other `getBy*` methods in `layouts.test.tsx` as well. Do I have to do so?

File: .changeset/tender-bears-confess.md (hunk 5 above)
Comment (txxxxc) [reply .changeset/tender-bears-confess.md]: fixed in [5e85eee](https://github.com/yamada-ui/yamada-ui/pull/976/commits/5e85eee5e127277d4a6652c3a0dca9068e670410)

**Comment by koralle** [comment]
@txxxxc 

> Thanks for the review, I just replaced the getBy* method with a findBy* method. For now, I just modified the code I added in the previous commit, and I would like to replace the other getBy* methods in layouts.test.tsx as well. Do I have to do so?

You only need to fix the new assertions you've added.   
The existing `getBy*` from before you created this PR will be corrected later in one go.

**Review by koralle — APPROVED** [review]

**Pull request merged**